### DISCUSSION TITLE
The third port in WT32-ETH01 does not work

### DISCUSSION THREAD

**Discussion by RUST1980** [forkineye/ESPixelStick#781] [General]
Hello. I am launching a project on WT32-ETH01 ESPixelStick-v4-1.0 (DDP)
I add objects to the ports in the Visualise tab..., then I Upload Output, everything loads, but when I go to ESPixelStick, only 2 ports are active, not 3 (((( And when I manually add a third port, the ethernet port stops working in ESPixelStick...
Pings disappear and ethernet does not work accordingly. I can only access ESPixelStick via Wi-Fi. 
Can you tell me what the problem might be? Or what is the correct PIN to use for the third?

**Comment by MartinMueller2003** [comment]
It sounds like there is a gpio misconfigured that is used by the ethernet device.

**Reply by RUST1980** [reply]
![Screenshot_4](https://github.com/user-attachments/assets/221466ce-76e2-4496-8b26-24c707c72923)
![Screenshot_5](https://github.com/user-attachments/assets/b060d094-491e-4ccc-88eb-9450f02c8eeb)

**Comment by MartinMueller2003** [comment]
The GPIO on port 5 is going to cause the Ethernet interface to stop working. You will need to set the GPIO for port 5 to an available pin.

**Reply by RUST1980** [reply]
Thank you so much, it's working!

**Comment by MartinMueller2003** [comment]
I found a 2nd overlap. I will be uploading an updated GPIO definitions file shortly.
